.gitignore

```diff
@@ -41,3 +41,6 @@ metastore_db/
 
 # Python stuff
 python/.mypy_cache/
+
+# mac specific
+.DS_Store
```

Comment:
Should this be in your global gitignore, instead of in every project-specific one?